src/components/PlayerInteractions/PlayerInteractions.jsx

```diff
@@ -4,28 +4,35 @@ import Prompt from '../shared/Prompt/Prompt';
 import Timer from '../Timer/Timer';
 import styles from './PlayerInteractions.module.css';
 import classNames from 'classnames/bind';
-import { setVoteCardTimer, setPlayCardTimer, setStorytellerTimer } from '../Timer/TimerActions';
+import { setVoteCardTimer, setPlayCardTimer, setStorytellerTimer, requestNextRound } from '../Timer/TimerActions';
 import { statusTypes } from '../../services/statusTypes';
 
 const cx = classNames.bind(styles);
 
 export class PlayerInteractions extends React.Component {
+    constructor(props) {
+        super(props);
+        this.requestNextRound = this.requestNextRound.bind(this);
+    }
+
+    requestNextRound() {
+        this.props.requestNextRound();
```

Comment:
This doesn't need to be a whole function, and you can then remove the constructor right?

Reply:
Oh I just realised Ronan already said this nevermind